docs/fsharp/tutorials/getting-started/getting-started-visual-studio.md

```diff
@@ -5,168 +5,147 @@ keywords: visual f#, f#, functional programming
 
-[!code-fsharp[Main](../../../../samples/snippets/fsharp/contour/snippet5.fs)]
+Another function, `main`, is defined, which is decorated with the `EntryPoint` attribute to tell the F# compiler that program execution should start there.  It follows the same convention as other [C-style programming languages](https://en.wikipedia.org/wiki/Entry_point#C_and_C.2B.2B), where command-line arguments can be passed to this function, and an integer code is returned (typically `0`).
 
-2. Another way to aggregate values is by using a list, as shown in the following code.
+It is in this function that we call the `square` function with an argument of `12`.  The F# compiler then assigns the type of `square` to be `int -> int` (that is, a function which takes an `int` and produces an `int`).  The call to `printfn` is a formatted printing function which uses a format string, similar to C-style programming languages, parameters which correspond to those specified in the format string, and then prints the result and a new line.
 
-[!code-fsharp[Main](../../../../samples/snippets/fsharp/contour/snippet7.fs)]
+## Running Your Code
 
-Add a new best friend to the list by using the "cons" operator (::). Note that the operation does not change the value of **bffs**. The value of **bffs** is immutable and cannot be changed.
+You can run the code and see results by pressing **ctrl-f5**.  This will run the program without debugging and allow you to see the results.  Alternative, you can click the **Debug** top-level menu item in Visual Studio and click **Start Without Debugging**.
 
-[!code-fsharp[Main](../../../../samples/snippets/fsharp/contour/snippet8.fs)]
+You should now see the following printed to the console window that Visual Studio popped up:
 
-Use **printfn** to display the lists. Function **printfn** shows the individual elements that are contained in structured values.
+```
+12 squared is 144!
+```
 
-[!code-fsharp[Main](../../../../samples/snippets/fsharp/contour/snippet9.fs)]
+Congratulations!  You've created your first F# project in Visual Studio, written an F# function printed the results of calling that function, and ran the project to see some results.
 
-3. You can view the results either by pressing CTRL+F5 or by selecting a section of the code and then pressing ALT+ENTER.
-<br />
+## Using F# Interactive
 
+One of the best features of the Visual F# tooling in Visual Studio is the F# Interactive Window.  It allows you to send code over to a process which allows you to call that code and see the result interactively.
```

Comment:
Too many allows you in one sentence. Can it be rewritten? 
Maybe "It allows you to send code over to a process where you can call that code and see the result interactively."